changelog/1.7.0.md

```diff
@@ -0,0 +1,12 @@
+# Scala.meta 1.7.0
+
+
+## Upgrade guide
+
+Contrib:
+- `equal` has been renamed to `isEqual` to avoid accidentally using `equals`,
+  which always compiles but does the wrong thing at runtime.
+- `isEqual` defaults to structual equality, so it's not longer necessary to
+  write `[Structurally]` in `a.isEqual[Structurally](b)`.
+- `(a: Subtype).isEqual(b: Supertype)` will no longer compile, either upcast
```

Comment:
Good point, you could argue though that the sibling case is a bug fix rather than a breaking change. Comparing sibling types will always be false in the structural equality case (although not in syntax 🤔 )